nixos/modules/services/misc/ntfy-sh.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.ntfy-sh;
+
+  settingsFormat = pkgs.formats.yaml { };
+in
+
+{
+  options.services.ntfy-sh = {
+    enable = mkEnableOption "[ntfy-sh](https://ntfy.sh), a push notification service";
```

Comment:
I do not know if `mkEnableOption` pass this to `lib.mdDoc`, maybe @pennae do know better than me about this?